docs/tutorials/inference/latent_distribution_test.ipynb

```diff
@@ -251,7 +251,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.0"
+   "version": "3.7.6"
```

Comment:
is this a necessary change?

Reply:
Strange, I didn't actually do that (not sure how it got there).  But it should be back to 3.7.0 now.